src/lib/tool_providers.ml

```diff
@@ -494,7 +506,7 @@ let default_toolkit
     star_tool ~host ~meta_playground;
     stringtie_tool ~host ~meta_playground;
     cufflinks_tools ~host ~meta_playground;
-    hisat_tool ~host ~meta_playground;
+    hisat_tool ~host ~meta_playground ~version:`V_0_1_6_beta;
```

Comment:
> Maybe have both versions in the toolkit, but the default tool should remain V2?

I thought have 2 versions of a single tool in the default kit seemed weird, I added V2 to the kit when I used it.